src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -411,12 +414,38 @@
           Returns="@(PackageFile)"
           DependsOnTargets="GetFiles;EnsureOOBFramework;$(VersionDependsOn)">
     <ItemGroup>
-      <PackageFile Include="@(File)" />
+      <!-- Include all files except source files. Sources need to be deduplicated. -->
+      <PackageFile Include="@(File)" Condition="'%(File.IsSourceCodeFile)'!='true'" />
       <PackageFile Condition="'%(PackageFile.PackageId)' == ''">
         <PackageId>$(Id)</PackageId>
         <PackageVersion>$(Version)</PackageVersion>
       </PackageFile>
 
+      <!-- Include Symbols -->
+      <PackageFileSymbol Condition="$([System.String]::new('%(PackageFile.Identity)').EndsWith('.dll'))"
```

Comment:
I'd prefer if we had packageLibs.targets do the symbol inclusion.  We can avoid complexity here if we just flow it through from the library project.